getting-started/setup-building.rst

```diff
@@ -414,27 +414,31 @@ OpenSSL which means that you will not be able to build the ``_ssl`` extension.
 One solution is to install these libraries from a third-party package
 manager, like Homebrew_ or MacPorts_, and then add the appropriate paths
 for the header and library files to your ``configure`` command.  For example,
+with **Homebrew**,
```

Comment:
```suggestion
for the header and library files to your ``configure`` command.

For example, with **Homebrew**, install the dependencies::
```

I might be wrong, but it's not clear separate whether `brew` invocations are required for each thing here, as it significantly complicates the structure and makes it somewhat unclear and harder to follow here.

Furthermore, even if so, It can still be simplified, and the introductions to each code block should be consistent with one another, rather than one calling out the Python version and the other the name of the step/tool, when in fact the former applies to both.

And its worth putting the introduction to the example on a separate line from this long and complex paragraph.